src/main/java/Command.java

```diff
@@ -0,0 +1,7 @@
+public abstract class Command {
+
+    abstract boolean isExit();
```

Comment:
There is only one command that leads to the program exiting, which is `bye`. As such, it would be better to have a Boolean variable `isExiting` that is initialised to `false` (by default) and only set to `true` in the ExitCommand.